python/tests/test_client/test_enum.py

```diff
@@ -117,6 +117,7 @@ def test_tag_enum() -> None:
 def test_app_enum() -> None:
     """Test `App` enum."""
     assert App.GITHUB == "GITHUB"
+    assert str(App.GITHUB) == "GITHUB"
```

Comment:
Consider adding more test cases to ensure the string representation works correctly in different scenarios:

```python
assert str(App.GMAIL) == "GMAIL"  # Test with different enum value
assert str(App("outlook")) == "OUTLOOK"  # Test with case-insensitive input
```